ydb/core/tx/replication/service/table_writer_impl.h

```diff
@@ -178,12 +178,16 @@ class TTablePartitionWriter: public TActorBootstrapped<TTablePartitionWriter<TCh
         return NKikimrServices::TActivity::REPLICATION_TABLE_PARTITION_WRITER;
     }
 
-    explicit TTablePartitionWriter(const TActorId& parent, ui64 tabletId, const TTableId& tableId)
-        : Parent(parent)
-        , TabletId(tabletId)
-        , TableId(tableId)
-    {
-    }
+    explicit TTablePartitionWriter(
+        const TActorId& parent,
+        ui64 tabletId,
+        const TTableId& tableId,
+        TChangeRecordBuilderContextTrait<TChangeRecord> builderContext)
+            : Parent(parent)
+            , TabletId(tabletId)
+            , TableId(tableId)
+            , BuilderContext(builderContext)
```

Comment:
Мы обычно наоборот пишем (по крайней мере в соседних файлах):
```suggestion
            const TActorId& parent,
            ui64 tabletId,
            const TTableId& tableId,
            TChangeRecordBuilderContextTrait<TChangeRecord> builderContext)
        : Parent(parent)
        , TabletId(tabletId)
        , TableId(tableId)
        , BuilderContext(builderContext)
```